Rc 1.1

dunglt91 opened the pull request.

Commit:
theme hs001 hs002 hs004

hs003 cac anh da doi nut roi nen em dang de mau xanh la cay
cac man hinh con lai em da lam xong

Commit:
fix conflic

khoahp commented:
@dunglt91 

**Không** được để file css trong JSP file

 +<style>
-   table tbody tr > td:last-child 
-   {
-       width:80px; 
-       vertical-align: middle !important;
-   }
  +</style>
  +
  <liferay-util:include page='<%=templatePath + "toptabs.jsp" %>' servletContext="<%=application %>" />
  <liferay-util:include page='<%=templatePath + "toolbar.jsp" %>' servletContext="<%=application %>" />

@@ -100,6 +108,7 @@

```
            //action column
            row.addJSP("center", SearchEntry.DEFAULT_VALIGN,"/html/portlets/dossiermgt/frontoffice/dossier_actions.jsp", config.getServletContext(), request, response);
```
-   
          %>  
      /liferay-ui:search-container-row

trungnt-ksnb commented:
@dunglt91 

Tại sao phải thay 

liferay-ui:icon bằng aui:button

dunglt91 commented:
On 06/14/2016 09:44 AM, Nguyễn Thành Trung wrote:

> @dunglt91 https://github.com/dunglt91
> 
> Tại sao phải thay
> 
> /aui:button
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub 
> https://github.com/VietOpenCPS/opencps/pull/445#issuecomment-225765543, 
> or mute the thread 
> https://github.com/notifications/unsubscribe/AQiIF-r8ioiZefvMT-98zN4-D-eQJhNEks5qLhWegaJpZM4I0Oed.
> 
> @anh trung: em trong thiết kế nó là nút anh ạ, @anh Khoa, em chưa tìm 
> được giải pháp làm thế nào để đưa CSS class vào 1 column dạng 
> row.addText, hoặc row.addJsp

Commit:
issue 444
sửa lỗi bị trùng số thành phần hồ sơ ở hồ sơ khác
thêm hàm tìm hiếm theo templateID và partNo
đã sửa xong

Commit:
dossier part
thêm giấy tờ kết nhiều kết quả (kiểu giấy tờ thứ 6)

Commit:
theme
sửa css
